packages/data/src/reports/utils.js

```diff
@@ -343,17 +344,30 @@ const getReportChartDataResponse = memoize(
  * @param  {string} options.dataType  'primary' or 'secondary'
  * @param  {Object} options.query     Query parameters in the url
  * @param  {Object} options.selector    Instance of @wordpress/select response
+ * @param  {Object} options.select    (Depreciated) Instance of @wordpress/select
  * @param  {Array}  options.limitBy   Properties used to limit the results. It will be used in the API call to send the IDs.
  * @param  {string}  options.defaultDateRange   User specified default date range.
  * @return {Object}  Object containing API request information (response, fetching, and error details)
  */
 export function getReportChartData( options ) {
 	const { endpoint } = options;
+	let reportSelectors = options.selector;
+	if ( options.select && ! options.selector ) {
+		deprecated(
+			'option.select is depreciated, please use option.selector',
```

Comment:
This results in the following message being output in the console:

`option.select is depreciated, please use option.selector is deprecated and will be removed in version 1.7.0. Note: You can pass the report selectors through option.selector now.`

So, I'd suggest just `option.select`, since that will result in a more coherent message. :-)